### DISCUSSION TITLE
Should I integrate or merge my scATAC datasets?

### DISCUSSION THREAD

**Discussion by Sun-storm** [stuart-lab/signac#1583] [Q&A] [answered]
Hi! 

I have the data from a 10X sequencing done on 4 brains tissue samples, but they were loaded in 4 different lanes and preprocessed separately. When I perform the signac tutorial for analyzing the data, I end up with this t-sne, which obviously represents the 4 different brains.

![louper_narcolepsy_signac_bcde_postNormalization](https://github.com/stuart-lab/signac/assets/93934490/4bcaaa90-0bcf-4cd0-bc90-1ab08eef8934)

I wish for them to overlap, but I'm unsure if I should integrate them or merge them, following the different tutorials. Merging seems to be what is suggested in the discussions, as integration seems to be reserved for ATAC with RNA, but I've been told to integrate, so I'm confused. I'm absolutely new to this world, and I don't know much, so please, be gentle if this should be a basic thing ^^

**Comment by YU-LIN96** [comment, marked as answer]
Check if you have created a combined peak. This is a big issue I had when processing scATAC. 
If you do not have a combined peak, that means there are sample-dependent features there and would cause a large bias at variable feature selection.
Try that first and see.
The integration tutorial does not include creating a common session as the Merge tutorial session does.

**Comment by Sun-storm** [comment]
I did as you suggested (by following the merge tutorial which taught me how to do the combined peaks) and I got a unified plot.

![image](https://github.com/stuart-lab/signac/assets/93934490/ac1380ce-ba4b-4181-9b91-1994ac322608)

Thank you :)
